doc/adr/0004-history-and-logging.md

```diff
@@ -0,0 +1,34 @@
+# app create - add new application to existing repository
+
+Date: 2024-12-13
+
+## Status
+
+Accepted
+
+## Context
+
+We're analysis our logging requirements. We want to have the history of commands that were run on corectl, 
```

Comment:
Not sure what 1st sentence is meant to mean here, but it could just be removed

Reply:
my typing doesn't always match what goes on in my spaghetti brain 😅